iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/cache/LoadCache.java

```diff
@@ -165,13 +170,21 @@ private void initRegionGroupHeartbeatCache(
             regionReplicaSets.forEach(
                 regionReplicaSet -> {
                   TConsensusGroupId regionGroupId = regionReplicaSet.getRegionId();
+                  boolean isStrongConsistency =
+                      (TConsensusGroupType.SchemaRegion.equals(regionGroupId.getType())
```

Comment:
make it a common function in conf related class?

Reply:
done